requirements.txt

```diff
@@ -1,5 +1,6 @@
-argparse==1.2.1
-cchardet
+six
+argparse
+cchardet==1.1.2
```

Comment:
Generally I don't think we want cchardet version locked. Good call on six and argparse though.